src/EnergyPlus/SingleDuct.cc

```diff
@@ -1791,8 +1791,11 @@ namespace SingleDuct {
 		//Do a check and make sure that the max and min available(control) flow is
 		//  between the physical max and min while operating.
 		SysInlet( SysNum ).AirMassFlowRateMaxAvail = min( Sys( SysNum ).AirMassFlowRateMax, Node( InletNode ).MassFlowRateMaxAvail );
-		SysInlet( SysNum ).AirMassFlowRateMinAvail = min( max( Node( OutletNode ).MassFlowRateMin, Node( InletNode ).MassFlowRateMinAvail ), SysInlet( SysNum ).AirMassFlowRateMaxAvail );
-
+		if ( Sys( SysNum ).SysType_Num == SingleDuctConstVolReheat ) { // AirTerminal:SingleDuct:ConstantVolume:Reheat
+			SysInlet( SysNum ).AirMassFlowRateMinAvail = min( max( Node( OutletNode ).MassFlowRateMin, Node( InletNode ).MassFlowRateMinAvail ), SysInlet( SysNum ).AirMassFlowRateMaxAvail );
+		} else { // #6173
+			SysInlet( SysNum ).AirMassFlowRateMinAvail = min( Node( OutletNode ).MassFlowRateMin, SysInlet( SysNum ).AirMassFlowRateMaxAvail );
+		}
```

Comment:
@lgu1234 can you explain why inlet node MinAvail isn't used for the VAV terminal unit?